src/cmd-push-container-manifest

```diff
@@ -122,11 +122,17 @@ def main():
             arch = manifest['platform']['architecture']
             if arch in map_arch:
                 arch = map_arch[arch]
-            buildmetas[arch][args.metajsonname] = {
-                'image': args.repo,
-                'digest': manifest['digest'],
-                'tags': args.tags
-            }
+            image = {
+                    'image': args.repo,
+                    'digest': manifest['digest'],
+                    'tags': args.tags
+                }
+            if buildmetas[arch].get(args.metajsonname, 'null') != 'null':
```

Comment:
by default the value of .get() for non existing key will be `None` so the following should work:
```suggestion
            if buildmetas[arch].get(args.metajsonname):
```